src/components/projects/projects-table/projects-table/projects-table.tsx

```diff
@@ -34,11 +34,52 @@ const ProjectsTable: React.FC<DisplayProjectProps> = ({ allProjects }: DisplayPr
   // TODO: Sort by wbsNum means 1.1.0 > 1.12.0 > 1.2.0, but desired is 1.1.0 > 1.2.0 > 1.12.0
   // TODO: Sort by duration means 12 > 2 > 4 > 5 > 9, but desired is 12 > 9 > 5 > 4 > 2
   const columns: ColumnDescription[] = [
-    { dataField: 'wbsNum', text: 'WBS #', align: 'center', sort: true },
-    { dataField: 'name', text: 'Name', align: 'left', sort: true },
-    { dataField: 'projectLead', text: 'Project Lead', align: 'left', sort: true },
-    { dataField: 'projectManager', text: 'Project Manager', align: 'left', sort: true },
-    { dataField: 'duration', text: 'Duration', align: 'center', sort: true }
+    {
+      headerAlign: 'center',
+      dataField: 'wbsNum',
+      text: 'WBS #',
+      align: 'center',
+      sort: true,
+      sortFunc: (a, b, order: SortOrder) => {
+        const a1 = a.split('.');
+        const b1 = b.split('.');
+        const len = Math.min(a1.length, b1.length);
```

Comment:
what suggests that the length of a1 would be different from b1?

Reply:
I wasn't 100% sure if there could be minor version updates (eg. 1.9 to 1.9.1), so I thought it would be good to account for it.

Reply:
No worries! WBS #s are not the same as semver. WBS #s are always 3 numbers separated by periods.